src/constants/index.ts

```diff
@@ -5,8 +5,8 @@ import { FullnameOption } from '@definition/settings';
 export const LANGUAGE_LIST = [
   { code: 'x', locale: 'de-DE', name: 'Deutsch' },
   { code: 'e', locale: 'en', name: 'English' },
-  { code: 's', locale: 'es-ES', name: 'español' },
-  { code: 's', locale: 'es-SSP', name: 'español (de España)' },
+  { code: 's', locale: 'es-ES', name: 'Español' },
+  { code: 's', locale: 'es-SSP', name: 'Español (de España)' },
```

Comment:
@rhahao is that OK to have the same lang code for both Spanish languages?

Reply:
@ux-git: this code property refers to the language code used when getting source materials from jw.org. Since Spanish (Spain) does not have their own materials, they are using Spanish.